llama_index/command_line/command_line.py

```diff
@@ -52,23 +56,36 @@ def handle_download_llama_dataset(
     print(f"Successfully downloaded {llama_dataset_class} to {download_dir}")
 
 
-def handle_question(
-    file_path: str,
-    question: str,
-    **kwargs: Any,
-) -> None:
-    assert file_path is not None
-    assert question is not None
+def default_rag_cli() -> RagCLI:
+    import chromadb
+
+    persist_dir = default_ragcli_persist_dir()
+    chroma_client = chromadb.PersistentClient(path=persist_dir)
+    chroma_collection = chroma_client.create_collection("default", get_or_create=True)
+    vector_store = ChromaVectorStore(
+        chroma_collection=chroma_collection, persist_dir=persist_dir
+    )
+    docstore = SimpleDocumentStore()
+    ingestion_pipeline = IngestionPipeline(
+        transformations=[SentenceSplitter(), OpenAIEmbedding()],
+        vector_store=vector_store,
+        docstore=docstore,
+        cache=IngestionCache(),
+    )
+    try:
+        ingestion_pipeline.load(persist_dir=persist_dir)
+    except FileNotFoundError:
+        pass
+
+    def persist_chroma(*args: list) -> None:
+        vector_store.persist(persist_dir)
```

Comment:
I don't think this is needed? `ChromaVectorStore` does not implement `persist()`, it just returns none

Reply:
gotcha, good catch!